api/jobs.yaml

```diff
@@ -483,6 +486,15 @@ definitions:
     enum:
       - text
       - list
+  BulkLabelField:
+    description: a DisplayField that can be bulk-editable via the UI
+    required:
+      - displayField
```

Comment:
It looks like you'll need to re-generate the models as well!

Reply:
done